docs/en/remote-flow.rst

```diff
@@ -358,8 +353,8 @@ The request and its parameters are defined in Section number 5 (Authorization Re
      - REQUIRED. Object with Credential format identifiers. See `OpenID4VP`_ Appendix B.
    * - `alg_values_supported`
      - OPTIONAL. Array of cryptographic suites supported. See `OpenID4VP`_ Appendix B.
-   * - `client_id_schemes_supported`
-     - RECOMMENDED. Array of Client Identifier schemes. Default is `entity_id`.
+   * - `client_id_prefixes_supported`
+     - RECOMMENDED. A non-empty array of the Client Identifier Prefixes that the Wallet Instance supports.  Valid values include ``openid_federation`` and ``x509_hash``; if omitted, the default is ``pre-registered``.
```

Comment:
we need an additional note for legacy RP using https URIs only, these would implicitly use the openid-federation client id scheme. The Wallet instances interacting with them would evaluate the trust with them collecting the trust chain about them, making them as pre-registered.